www/templates/diary/list.html

```diff
@@ -33,17 +33,18 @@
         MUST test on android first to make sure that the scrolling works.
         -->
 
-		<div collection-repeat="tripgj in data.currDayTripWrappers" item-height="differentCommon(tripgj)? '348px':'308px'">
+		<div collection-repeat="tripgj in data.currDayTripWrappers" item-height="differentCommon(tripgj)? '413px':'365px'">
            <div class="start-time-tag">{{localTimeString (tripgj.data.properties.start_local_dt)}}</div>
             <div  style="padding-left: 19%;">
 
-			 <ion-item style="background-color: transparent;" id="no-border" class="list-item" href="#/root/main/diary/{{tripgj.data.id}}">
```

Comment:
can't you leave the href on the non-button div? Just curious so I can understand how href works.